lib/hammer_cli/main.rb

```diff
@@ -39,7 +39,7 @@ class MainCommand < AbstractCommand
     option ["--interactive"], "INTERACTIVE", _("Explicitly turn interactive mode on/off"),
       :format => HammerCLI::Options::Normalizers::Bool.new,
       :context_target => :interactive
-
+    option ["--no-headers"], :flag, _("Hide headers from output")
```

Comment:
The option should also be defined in `bin/hammer` (like e.g.https://github.com/theforeman/hammer-cli/blob/master/bin/hammer#L31) otherwise the pre-parser there fail on unknown option and will not do its job:
```
$  hammer --no-headers -s https://fqdn  hostgroup list
Could not load the API description from the server: 
Failed to open TCP connection to foreman.my.lan:80 (getaddrinfo: Name or service not known)
  - is the server down?
```

Reply:
Updated